libqtile/backend/x11/xcbq.py

```diff
@@ -619,6 +676,9 @@ def paint(self, screen, image_path, mode=None):
             logger.error('Wallpaper: %s' % e)
             return
 
+        max_depth = max(self.default_screen.visuals)
+        _, root_visual = self.default_screen.visuals[max_depth]
```

Comment:
It's this bit that errors for me, because `self.default_screen.visuals` is just a list of visuals and so it can't be unpacked into two parts

Reply:
Yup - this is wrong. Fixed in latest commit.